internal/graph/generated.go

```diff
@@ -20,6 +20,10 @@ type AvailableUpdate struct {
 	Sha256   string `json:"sha256"`
 }
 
+type CheckRuntimeLastUsedResponse struct {
+	Times map[string]interface{} `json:"times"`
```

Comment:
Please have this either return a string value map, or ideally even a datetime if graphql supports it.

If this is too challenging to handle on the protocol level we could dial it up to the model. But we shouldn't be passing interface{} values around if we can help it.

Reply:
graphql's "Map" scalar is a `map[string]interface{}` so there was no easy way around it from what I could see. Anyway, I've changed the schema for the better.